prow/plugins.yaml

```diff
@@ -206,7 +202,10 @@ slack:
         release-1.14:
             - aleksandra-malinowska # 1.14 patch release team
             - tpepper # 1.14 patch release team
-            - hoegaarden # 1.14 branch manager
+            - hoegaarden # 1.14 patch release team
+        release-1.15:
+            - bubblemelon # 1.15 branch manager
```

Comment:
I'm not sure if about this since @Bubblemelon has a different username on slack

Reply:
These are github usernames, not slack usernames.

Reply:
Thanks! So the mention won't work?